cts-java/src/test/java/org/ga4gh/cts/api/reads/ReadsSearchIT.java

```diff
@@ -39,33 +39,22 @@
      */
     @Test
     public void searchReadsWithUninterestingRangeProducesZeroReads() throws AvroRemoteException {
```

Comment:
Didn't Sherlock Holmes have something to say about a dog that didn't bark? ie. it's not that the range is uninteresting (it may be fascinating!) it's just that the dataset we're testing has no reads overlapping it, right? So, perhaps a new name is in order:

`searchRangeWithNoReadsReturnsZeroResults()`